src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,334 @@
   </div>
 </div>
 
+{% if paymentOption is defined %}
+<div class="section">
+  <div class="section-heading">
+    <strong>When the defendant will pay</strong>
+  </div>
+    <div class="section-body">
+      <table>
+        <tr class="no-border">
+          <td><strong>When will they pay:</strong></td>
+          <td>
+            {{ paymentOption }}
+          </td>
+        </tr>
+
+        {% if repaymentPlan is defined %}
+        <tr class="no-border">
+          <td><strong>Regular payments of:</strong></td>
+          <td>
+            {{ repaymentPlan.instalmentAmount }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Frequency of payments:</strong></td>
+          <td>
+            {{ repaymentPlan.paymentSchedule }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Make the first payment on:</strong></td>
+          <td>
+            {{ repaymentPlan.firstPaymentDate }}
+          </td>
+        </tr>
+        {% if statementOfMeans is defined %}
+        <tr class="no-border">
+          <td><strong>Explain why you can't pay the full amount now:</strong></td>
+          <td>
+            {{ statementOfMeans.reason}}
+          </td>
+        </tr>
+        {% endif %}
+        {% endif %}
+      </table>
+    </div>
+</div>
+{% endif %}
+
+{% if paymentOption == 'By instalments' or  paymentOption == 'By a set date' %}
+<div class="section">
+  <div class="section-heading">
+    <strong>The defendant's financial details</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr class="no-border">
+        <td><strong>Where are they living?</strong></td>
+        <td>
+          {{ residence.typeDescription }}
+        </td>
+      </tr>
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Children:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Do any children live with them?</strong></td>
+        <td>
+        {% if children is defined and children is not empty %}
```

Comment:
consider extracting these 
```
if var is defined and var is not empty then
yes / no
```

to a macro
REC

Reply:
That is a good idea @timja-kainos